core-bundle/src/Fragment/FragmentHandler.php

```diff
@@ -57,6 +69,25 @@ public function __construct(ContainerInterface $renderers, BaseFragmentHandler $
         parent::__construct($requestStack, [], $debug);
     }
 
+    public function pushStrategy(ResponseCacheStrategy $strategy): void
+    {
+        if ($this->currentStrategy) {
+            $this->strategyStack[] = $this->currentStrategy;
+        }
+
+        $this->currentStrategy = $strategy;
+    }
+
+    public function popStrategy(): void
+    {
+        if (0 === \count($this->strategyStack)) {
```

Comment:
I think this can be reduced to just `$this->currentStrategy = array_pop($this->strategyStack);` (see https://3v4l.org/fGQGb).

Reply:
Changed in 6dac1ef004af2f2ecfa50f7c0d383d5d4adf8119